solutions/gold/cses-1193.mdx

```diff
@@ -198,5 +198,73 @@ public class cses1193 {
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+from collections import deque
+
+MAX_N = 1000
+step_dir = "URDL"
+vis = [[False for _ in range(MAX_N)] for _ in range(MAX_N)]
+
+# previous_step stores the previous direction that we moved in to arrive at this cell
+previous_step = [[-1 for _ in range(MAX_N)] for _ in range(MAX_N)]
+
+# 0 = up, 1 = right, 2 = down, 3 = left
+dx = [-1, 0, 1, 0]
+dy = [0, 1, 0, -1]
```

Comment:
the constants thing applies to these as well
also make their names uppercase since they're constants (including step-dir)

Reply:
done